app/views/layouts/base.html.erb

```diff
@@ -32,6 +32,9 @@
       <% end %>
     </script>
     <%= javascript_include_tag "locale/#{FastGettext.locale}/app" %>
+    <% for plugin in ::Foreman::Plugin.all do %>
```

Comment:
Technically not all plugins have translations, so it may end up resulting in HTTP 404 calls.

Reply:
Changed, even though we might want to revisit it once we extend the plugin dsl.